migrations/versions/f69f31bbf7d2_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: f69f31bbf7d2
+Revises: 44d4df0fe283
+Create Date: 2022-06-11 09:51:06.770739
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f69f31bbf7d2'
+down_revision = '44d4df0fe283'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('review', sa.Column('dive_shop_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'review', 'dive_shop', ['dive_shop_id'], ['id'])
```

Comment:
Where it says "None" we need to put a name. i think we can do `review__dive_shop_foreign_key`.